### DISCUSSION TITLE
Sign up for GitHub’s Customer Research Panel!

### DISCUSSION THREAD

**Discussion by tuves** [community/community#38696] [Announcements]
Once you sign up you’ll have a chance to win a coveted neon Octocat sign 😎

If you end up participating in research, you’ll have the opportunity to give feedback on early designs, get access to betas, and inform our product direction! Plus, we’ll compensate you for your time 🤑

[Click here to sign up!](https://github.co/GH_Research_Panel)

![IMG_7737](https://user-images.githubusercontent.com/2140740/199746104-7b91232e-f8a4-44e1-8281-29d542f14ef6.jpg)

For more information about participating in GitHub Research studies, [click here](https://cxr.github.com/).

### Sweepstakes Terms & Conditions

After signing up for the Research Panel at https://github.co/GH_Research_Panel, you will be entered into a sweepstakes to be conducted on November 30, 2022, for one limited edition neon Octocat sign. 

The drawing will be conducted by the GitHub Customer Research Team in San Francisco, CA. The chance of winning a prize is approximately 1 in 300. The winner will be notified immediately by email and be asked for a mailing address to send the prize.

NO PURCHASE NECESSARY. Open only to legal residents of the 50 U.S. and D.C., must be 18 or older. Sweepstakes ends November 30, 2022. For Official Rules, see https://github.co/Research-Panel-Sweepstakes.

**Comment by Andre601** [comment]
> NO PURCHASE NECESSARY. Open only to legal residents of the 50 U.S. and D.C., must be 18 or older. Sweepstakes ends November 30, 2022. For Official Rules, see https://github.co/Research-Panel-Sweepstakes.

When understanding this right can or should I not register? I live in switzerland, not the USA...

Also, there's a typo in the URL. It's `.co` instead of `.com` ;)
(This is also the case in the form)

**Reply by samus-aran** [reply]
Hi @Andre601 👋 

Great question! Registration for GitHub's Customer Research Panel is open to all 🌏 It's not limited to only U.S residents. Sorry for that misunderstanding. We would love for you to sign up ❤️ 

However the sweepstakes to win the limited edition neon Octocat sign is only applicable to residents of the 50 U.S. and D.C. Hope that clarifies everything.

The URL `.co` is accurate, both are acceptable! 👍

**Reply by Andre601** [reply]
Thank you for the clarifications.

To be quite honest, this is kinda bad from my point of view. Given the fact that US citizens will have the chance of getting some free swag for participation, is not a good thing.
Because with that there will be people who only participate for having the chance of getting this free swag in the end. Meanwhile people coming from Europe, Asia or other places outside the US (Like myself) will most likely either not participate (As the line wasn't quite clear as you can guess from my previous questions) or participate with the goal of actively helping in research and beta-testing features.

So, in short: People participating from the US may have a bias due to the price they can get, which can (and probably will) influnce your reasearch and beta-testing attempts in a negative way.

That's obviously my point of view and people may actually really participate to help GitHub improve... You never know how someone thinks and behaves after all :)

**Reply by sbn2030** [reply]
WHT
966583332411

I want a service provider

**Reply by Johnconley24** [reply]
Thank you for the clarifications.

To be quite honest, this is kinda bad from my point of view. Given the fact that US citizens will have the chance of getting some free swag for participation, is not a good thing.
Because with that there will be people who only participate for having the chance of getting this free swag in the end. Meanwhile people coming from Europe, Asia or other places outside the US (Like myself) will most likely either not participate (As the line wasn't quite clear as you can guess from my previous questions) or participate with the goal of actively helping in research and beta-testing features.

Pills for male enhancement to improve your sexual performance.

[Sildenafil](https://www.ultrapharmrx.com/)

**Comment by Badrulsales** [comment]
[baca saya](https://github.com/Badrulsales/readme) Awam
👋Hai, saya @Badrulsales -👀Saya berminat dalam ... -🌱Saya sedang belajar ... -💞️saya sedang mencari...
Lesen Apache 2.0 Dikemas kini1 minit yang lalu
[Badrulsales](https://github.com/Badrulsales/Badrulsales) Awam
Konfigurasi fail untuk profil GitHub saya.
[konfigurasi](https://github.com/topics/config)
[github-config](https://github.com/topics/github-config)

**Comment by sbn2030** [comment]
Cari pengenalan kepada telefon pintar dan beberapa perkhidmatan perisian

بتاريخ ١٢‏/١١‏/٢٠٢٢ ٩:٤٢ م، كتب Badrulsales ***@***.***>:

baca saya<https://github.com/Badrulsales/readme> Awam
👋Hai, saya @Badrulsales<https://github.com/Badrulsales> -👀Saya berminat dalam ... -🌱Saya sedang belajar ... -💞️saya sedang mencari...
Lesen Apache 2.0 Dikemas kini1 minit yang lalu
Badrulsales<https://github.com/Badrulsales/Badrulsales> Awam
Konfigurasi fail untuk profil GitHub saya.
konfigurasi<https://github.com/topics/config>
github-config<https://github.com/topics/github-config>

**Comment by Shofiqsarkar** [comment]
> Once you sign up you’ll have a chance to win a coveted neon Octocat sign 😎
> 
> If you end up participating in research, you’ll have the opportunity to give feedback on early designs, get access to betas, and inform our product direction! Plus, we’ll compensate you for your time 🤑
> 
> [Click here to sign up!](https://github.co/GH_Research_Panel)
> 
> ![IMG_7737](https://user-images.githubusercontent.com/2140740/199746104-7b91232e-f8a4-44e1-8281-29d542f14ef6.jpg)
> 
> For more information about participating in GitHub Research studies, [click here](https://cxr.github.com/).
> 
> ### Sweepstakes Terms & Conditions
> After signing up for the Research Panel at https://github.co/GH_Research_Panel, you will be entered into a sweepstakes to be conducted on November 30, 2022, for one limited edition neon Octocat sign.
> 
> The drawing will be conducted by the GitHub Customer Research Team in San Francisco, CA. The chance of winning a prize is approximately 1 in 300. The winner will be notified immediately by email and be asked for a mailing address to send the prize.
> 
> NO PURCHASE NECESSARY. Open only to legal residents of the 50 U.S. and D.C., must be 18 or older. Sweepstakes ends November 30, 2022. For Official Rules, see https://github.co/Research-Panel-Sweepstakes.

**Comment by butlerJ28** [comment]
i want to join pls gusy

**Comment by dhrubasaha08** [comment]
@butlerJ28 Follow the above instructions

**Comment by janesroberta** [comment]
ambos são aceitáveis!👍

**Comment by BaseMax** [comment]
Let me know if I can do anything.

**Comment by sinalalebakhsh** [comment]
I can't understand this

**Comment by sinalalebakhsh** [comment]
I'm am in Iran and can't to signup.
Thanks

**Reply by BaseMax** [reply]
Hi Sina, Glad to meet you here. There is anything I can help you with?

**Reply by BaseMax** [reply]
Dear Sina, if you are going to fill out the research form to get $25 USD, I can say the deadline passed and I don't think the team is following it now. It was for Nov of 2022.

For more details, feel free to check https://github.com/github/Research-Panel-Sweepstakes/blob/1128c66f3bb6be5865a34eda3886ad128562712d/GitHub%20Universe%20Customer%20Research%20Sweepstakes%20Official%20Rules.docx.pdf PDF file.

If you have any other questions, I'm here to help if I can.

Your friend,
M.

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
